STRATEGIC-INITIATIVES.md

```diff
@@ -0,0 +1,46 @@
+# Strategic Initiatives
+
+At any one time the Node.js project has a number of strategic initiatives
+underway.  With the less-tangible approach to the work that the Community 
+Committee does, having a way to track this work is important.
+
+For each strategic initiative, the Community Committee's goal is to have a
+single point of reference - a champion - that understands what's happening
+with the initiative to the fullest extent posssible, and can report on the
+initiative as needed.
+
+A review of the initiatives will be a standing item on the Community Committee agenda (even
+if the update is 'nothing new') as a way to ensure they are active
+and have the support needed.
+
+# Current Initiatives
+
+| Initiative         | Champion                        | Links                                                            |
+|--------------------|---------------------------------|------------------------------------------------------------------|
+| NodeTogether       | Rachel White                    | https://github.com/nodejs/community-committee/issues/63          |
+| User Feedback      | Michael Dawson                  | https://github.com/nodejs/user-feedback                          |
+| Community Events   | PatrickHeneise                  | https://github.com/nodejs/community-events                       |
+| Node.js Collection | Tierney Cyren                   | https://github.com/nodejs/nodejs-collection                      |
+| i18n               | Rachel White                    | https://github.com/nodejs/community-committee/issues/114         |
+| Education          | Tracy Hinds                     | https://github.com/nodejs/education                              |
+| Evangelism WG      | Tierney Cyren                   | https://github.com/nodejs/evangelism                             |
+| Office Hours       | Tierney Cyren                   | https://github.com/nodejs/community-committee/issues/157         |
```

Comment:
I would love to help this become a reality. With @nodejs/user-feedback we hold meetings every 2 weeks. We are dedicating meetings every couple months to run external facing feedback sessions instead of a organizational team meeting. At present, this is aspirational, not strategic.

Reply:
I am planning to do an Office Hours session either this weekend or just after the holiday - plan is to do review / cleanup of the repo and Q&A as questions arise.